units/user@.service.in

```diff
@@ -11,7 +11,7 @@
 Description=User Manager for UID %i
 Documentation=man:user@.service(5)
 BindsTo=user-runtime-dir@%i.service
-After=user-runtime-dir@%i.service dbus.service systemd-oomd.service
+After=systemd-logind.service user-runtime-dir@%i.service dbus.service systemd-oomd.service
```

Comment:
Hmm, I think this should also be added to `user-runtime-dir@.service`. That also acquires some properties from logind, and this makes things consistent wrt the commit that dropped the dep on user-session.service